components/director/internal/domain/fetchrequest/service.go

```diff
@@ -0,0 +1,74 @@
+package fetchrequest
```

Comment:
I've found some unused code related to fetchrequests in application domain (service), could you remove it?